test/models/task_comment_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class TaskCommentTest < ActiveSupport::TestCase
```

Comment:
add rspec right now or don't commit test files for now. Delete them after autogeneration. The default picked library is minitest and we don't use it.